api/office/schema.py

```diff
@@ -22,6 +23,10 @@ class Arguments:
 
     @Auth.user_roles('Admin')
     def mutate(self, info, **kwargs):
+        existing_office = OfficeModel.query.filter_by(
```

Comment:
This would only check for one error, the conflict, it would be great to use SQLALCHEMY EXCEPTIONS and overide the messages, as such we will catch lots of errors includuing db connection, conflicts, invalid request, invalid arguments etc, this is better used in a try except block.
we have 2 options:
-Use a try except block in the mutation
-Build a context manager which would encapsulate the try except block with sqlalchemy exceptions

Reply:
Well noted @shemogumbe . Working on it.